sdk/src/main/java/com/phunware/parkassist/ParkAssistSDK.java

```diff
@@ -101,30 +101,28 @@ public void searchPlates(String partialPlate, Location location,
             paramMap.put(PARAMS_LATITUDE, latitude);
             paramMap.put(PARAMS_LONGITUDE, longitude);
             paramMap.put(PARAMS_PLATE, partialPlate);
-            ParkAssistHttpClient.get(generatePlateGetUrl(SEARCH_ENDPOINT, paramMap),
-                    new JsonHttpResponseHandler() {
-
-                @Override
-                public void onSuccess(int statusCode, Header[] headers, JSONArray response) {
-                    Log.d(TAG, "Status code: " + statusCode);
-                    List<PlateSearchResult> results = new LinkedList<>();
-                    for (int i = 0; i < response.length(); i++) {
-                        try {
-                            results.add(i, new PlateSearchResult(response.getJSONObject(i)));
-                        } catch (JSONException e) {
-                            callback.onFailed(e);
-                            return;
+            httpClient.getJSON(generatePlateGetUrl(SEARCH_ENDPOINT, paramMap),
+                    new ParkAssistNetworkingInterface.ParkAssistJSONResponseInterface() {
+
```

Comment:
204 (no content) will pass a blank JSONObject to the success block. if the bytes that come back in the response are null and the status code isn't 204 then we will likely end up in the failure block. However 0 responses should still return an empty array instead of null (according to API documentation) so I am comfortable with this as-is.